changelog/20285.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+openapi: Small fixes for OpenAPI display attributes
```

Comment:
[nit]
would even include what you did here
```suggestion
openapi: Small fixes for OpenAPI display attributes. Changed "log-in" to "login"
```

Reply:
Good idea